instamatic/processing/ImgConversion.py

```diff
@@ -32,9 +32,9 @@ def rotation_axis_to_xyz(rotation_axis, invert=False, setting='xds'):
     rot_z = 0
 
     if setting == 'dials':
-        return rot_x, -rot_y, rot_z
-    elif setting == 'xds':
         return rot_x, rot_y, rot_z
+    elif setting == 'xds':
+        return rot_x, -rot_y, rot_z
```

Comment:
I agree this is something to be looked at.

Just off the bat, I'm happy discard this change (accept your suggestion) and continue this discussion in a private email conversation or so, since it's maybe more to do with a misunderstanding on my part and less to do with the code in instamatic/edtools.

Unless I'm missing something obvious (could easily happen to me) I actually based this change on what is written in [https://github.com/instamatic-dev/edtools/blob/master/edtools/find_rotation_axis.py](https://github.com/instamatic-dev/edtools/blob/master/edtools/find_rotation_axis.py) where these _are_ flipped.. This also agrees with what is for example also stated in the paper by [Heidler et al. 2019](https://doi.org/10.1107/S2059798319003942): $R_{init} = \cos(\alpha), \sin(\alpha), 0$
Copy/pasted the code from the edtools script:
```
    if setting == 'dials':
        return rot_x, -rot_y, rot_z
    elif setting == 'xds':
        return rot_x, rot_y, rot_z
    else:
        raise ValueError("Must be one of {'dials', 'xds'}")
```
here is why I added the change: running edtools.find_rotation_axis.py (thanks for the reminder!) on one of my datasets gives me
```
Rotation axis found: 114.80 deg. / 2.004 rad.
 - Instamatic (config/camera/camera_name.yaml)
    rotation_axis_vs_stage_xy: 2.004
 - XDS
    ROTATION_AXIS= -0.4195 0.9078 0.0000
 - XDS (opposite rotation)
    ROTATION_AXIS= 0.4195 -0.9078 0.0000
 - DIALS
    geometry.goniometer.axes=-0.4195,-0.9078,0.0000
 - DIALS (opposite rotation)
    geometry.goniometer.axes=0.4195,0.9078,0.0000
 - PETS (.pts)
    omega 114.80
 - RED (.ed3d)
    ROTATIONAXIS    114.8000
```
whereas running `process_tvips.py` (with rotations axis in `cRED_log.txt` set to 115) results in an XDS.INP file with the line
```
ROTATION_AXIS= -0.4258 -0.9048 0.0000
```
Processing this dataset further with XDS only works well when using `ROTATION_AXIS= -0.4195 0.9078 0.0000
` and gives (the reasonable for this sample) unit cell parameters of "14.32 18.15 5.29 90 90 90"), in contrast to the alternative rot_axis setting, where XDS only throws an error about insufficient percentage of indexed reflections (< 25%). This led me to the conclusion that the "true" rot_axis should be ~115°

PS: to confuse myself even more, PETS2 gives a rotation axis of approx 254.3° (360-114) for this dataset although I have to admit I don't exactly know how they describe the geometry (for cRED) internally and I will need to look into it more. But this can be dealt with in PETS itself very easily, as it has both a global search and refining step for the tilt axis built in, which does basically (exactly?) the same as the edtools script.

Reply:
Haha, yeah, this was a huge HUGE pain to figure out all the rotation axes. I can't even remember how many times I have switched the numbers around to get this right.

That's why I wrote `find_rotation_axis.py` to just calculate the best rotation axis, and I've been using that as my source of truth. 

So based on that this seems like a bug, thanks for figuring it out! 👍

PS. To give a bit of history, I was so impressed by how Lukas calculated the rotation axis in PETS when I first saw it that I wrote my own version in Python to understand how it works. It's not a coincidence they work virtually the same 😅 In case you are interested, the original algorithm is in this book: Gorelik et al. (Introduction to ADT/ADT3D. In Uniting Electron Crystallography and Powder Diffraction (2012), 337-347)